ci/cpu/dask-cudf/build_dask_cudf.sh

```diff
@@ -6,5 +6,10 @@ function logger() {
   echo -e "\n>>>> $@\n"
 }
 
+# If nightly build, append current YYMMDD to version
```

Comment:
This should not be in this file, it should be in ci/cpu/build.sh that way it is set once

Reply:
Will make the changes myself and change the PR to bring all the scripts in-line